packages/api/src/external/fhir/document/index.ts

```diff
@@ -163,22 +168,40 @@ export function idToFHIR(id: DocumentIdentifier): Identifier {
  * @param resource
  * @returns
  */
-function convertToFHIRResource(resource: Contained): Resource | undefined {
-  if (resource.resourceType === ResourceType.Patient && resource.id) {
+function convertToFHIRResource(resource: Contained, patientId: string): Resource | undefined {
+  if (resource.resourceType === "Patient" && resource.id && !resource.id.includes(patientId)) {
+    capture.message(`Found a matching Patient resource in CW docRef.contained`, {
```

Comment:
wouldn't something like `Found a Patient resource with differing id...` be more accurate?

Reply:
Oh, yes, of course..